req.raw.cf types broken in 3.7.4

### What version of Hono are you using?

3.7.4

### What runtime/platform is your app running on?

Cloudflare Workers

### What steps can reproduce the bug?

```javascript
import { Hono } from 'hono';

const app = new Hono();

app.get('/country', async (context) => {
	const country = context.req.raw?.cf?.country ?? '';
	context.text(`Country: ${country}`)
});

export default app;

```

### What is the expected behavior?

In version 3.7.3, `context.req.raw?.cf?.country ?? ''` does not have any issues. TypeScript is happy.

### What do you see instead?

In version 3.7.4, `context.req.raw?.cf?.country ?? ''`, the following TypeScript error is thrown:

```
src/index.ts:10:35 - error TS2339: Property 'cf' does not exist on type 'Request'.

10  const country = context.req.raw?.cf?.country ?? ''
```

### Additional information

_No response_

--- a/deno_dist/context.ts
+++ b/deno_dist/context.ts
@@ -1,9 +1,6 @@
-/// <reference lib="es2022" />
-/// <reference lib="webworker" />
-// We need these triple slashes to correctly refer to `FetchEvent` in Deno.
-
 import type { HonoRequest } from './request.ts'
 import type { Env, NotFoundHandler, Input, TypedResponse } from './types.ts'
+import { FetchEvent } from './types.ts'
 import type { CookieOptions } from './utils/cookie.ts'
 import { serialize } from './utils/cookie.ts'
 import type { StatusCode } from './utils/http-status.ts'

--- a/deno_dist/types.ts
+++ b/deno_dist/types.ts
@@ -592,3 +592,5 @@ export abstract class FetchEventLike {
   abstract passThroughOnException(): void
   abstract waitUntil(promise: Promise<void>): void
 }
+
+export abstract class FetchEvent extends FetchEventLike {}

--- a/src/adapter/vercel/handler.ts
+++ b/src/adapter/vercel/handler.ts
@@ -1,6 +1,7 @@
 // @denoify-ignore
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import type { Hono } from '../../hono'
+import type { FetchEvent } from '../../types'
 
 export const handle = (app: Hono<any, any, any>) => (req: Request, requestContext: FetchEvent) => {
   return app.fetch(req, {}, requestContext as any)

--- a/src/context.ts
+++ b/src/context.ts
@@ -1,9 +1,6 @@
-/// <reference lib="es2022" />
-/// <reference lib="webworker" />
-// We need these triple slashes to correctly refer to `FetchEvent` in Deno.
-
 import type { HonoRequest } from './request'
 import type { Env, NotFoundHandler, Input, TypedResponse } from './types'
+import { FetchEvent } from './types'
 import type { CookieOptions } from './utils/cookie'
 import { serialize } from './utils/cookie'
 import type { StatusCode } from './utils/http-status'

--- a/src/types.ts
+++ b/src/types.ts
@@ -592,3 +592,5 @@ export abstract class FetchEventLike {
   abstract passThroughOnException(): void
   abstract waitUntil(promise: Promise<void>): void
 }
+
+export abstract class FetchEvent extends FetchEventLike {}

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -10,10 +10,6 @@
     "skipLibCheck": false,
     "noUnusedLocals": false,
     "noUnusedParameters": false,
-    "lib": [
-      "ES2022",
-      "WebWorker"
-    ],
     "types": [
       "jest",
       "node"

